challenges/desafio6.js

```diff
@@ -0,0 +1,9 @@
+db.produtos.updateMany(
+  { nome: { $in: ["Big Mac", "Quarteirão com Queijo"] } },
+  { $addToSet: { ingredientes: "bacon" } },
```

Comment:
```suggestion
  { $push: { ingredientes: "bacon" } },
```
O $addToSet apenas garante que não há elementos duplicados sendo adicionados, e ele não garante uma ordem específica ao modificar o array. Já o $push sempre adicionará elementos no final do array (que é o que o exercício pede), ao penos que vc especifique a ordem com o $position. Então aqui acho que o $push faz mais sentido, o que acha?